connaisseur/util.py

```diff
@@ -130,3 +131,13 @@ def get_kube_version():
     regex = r"v(\d)\.(\d{1,2})\.(.*)"
     match = re.match(regex, version)
     return match.groups() if match else ("0", "0", "0")
+
+
+def feature_flag_on(flag: str):
+    env = os.environ.get(flag)
+
+    # only false if env == None
+    if not env:
```

Comment:
Before there were defaults for some flags, i.e. `os.environ.get("AUTOMATIC_CHILD_APPROVAL_ENABLED", "1")`. Why is this not a breaking change for these situations?

Reply:
Both in the sense of "before undefined defaulted to something instead of boom" and technically for "before 'y' meant False, now it's True". The later should be fine, if you highlight it in the v3 changelog as breaking. The first seems unnecessary